requirements.txt

```diff
@@ -1,3 +1,4 @@
 qiskit>=0.12
 PennyLane>=0.7.0
 numpy
+networkx==2.3
```

Comment:
Note to self: "Pinning networkx version to 2.3 such that tests pass for Python 3.5 as required by qiskit-terra"